feature/gribi/ate_tests/base_hierarchical_route_installation_test/README.md

```diff
@@ -6,35 +6,48 @@ Validate IPv4 AFT support in gRIBI with recursion.
 
 ## Procedure
 
-*   Connect ATE port-1 to DUT port-1 and ATE port-2 to DUT port-2.
-*   Establish gRIBI client connection with DUT using default parameters.
-*   Using gRIBI Modify RPC install the following IPv4Entry set:
-    *   198.51.100.0/24 to NextHopGroup containing one NextHop, specified to be
-        203.0.113.1/32
-    *   203.0.113.1/32 to NextHopGroup containing one NextHop specified to be
-        the address of ATE port-2.
-*   Forward packets between ATE port-1 and ATE port-2 (destined to
+**Topology**
+
+*  Connect ATE port-1 to DUT port-1 and ATE port-2 to DUT port-2.
+    *   TODO: create a non-default VRF (VRF-1) that includes DUT port-1.
+
+**Validate hierarchical resolution**
+
+1.  Establish gRIBI client connection with DUT.
+1.  TODO: Use gRIBI Modify RPC to install entries per the following order, and ensure FIB ACK is received for each of the AFTOperation:
```

Comment:
Should we add the Persistence mode to make it more specific or is fine without specifying the persistence mode for gRIBI?

Reply:
We're doing PRESERVE + optionally DELETE. See #447

Reply:
Thanks. I'll close this comment and handle the merge with #447 later.